ihatemoney/tests/api_test.py

```diff
@@ -0,0 +1,773 @@
+import base64
+import datetime
+import json
+import unittest
+
+from ihatemoney.tests.common.help_functions import em_surround
+from ihatemoney.tests.common.ihatemoney_testcase import IhatemoneyTestCase
+
+project_name = "raclette"
+
+
+class APITestCase(IhatemoneyTestCase):
+
+    """Tests the API"""
+
+    def api_create(self, name, id=None, password=None, contact=None):
+        id = id or name
+        password = password or name
+        contact = contact or f"{name}@notmyidea.org"
```

Comment:
In general, I tend to prefer having the tests easy to read, so relying on a variable name here seems weird to me.

Reply:
Also, if we chose to go with the variable name, I believe it should be something we can change by subclassing the tests. So maybe put it in a constant inside the class?

Reply:
Hello, thanks for taking the time of reviewing this PR :D

I'm not really sure of understanding the changes you're asking me. Could you clarify with a little example ?

This part of the tests was already like that when I separated in several files. I've only added the
`project_name = "raclette"` and replace it where it seems adequate to do so.

But If you explain it to me, I'll be glad to help

Reply:
So, my idea was to revert the `project_name = "raclette"`` part, because I believe it's just adding a layer of redirection and makes the tests harder to read in my opinion. What do you think about this?

Reply:
Yeah, i was not sure about the result, I agree it will be better without it, I will fix this when I have some free time next days. thanks 👍

Reply:
@indatwood as you requested I've removed the variable *project_name*, so now the tests are unchanged only separated in many files :)